.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
   integration-test:
     name: integration-tests ${{ matrix.builder }} / ${{ matrix.arch }}
-    runs-on: ${{ matrix.arch == 'arm64' && 'pub-hk-ubuntu-22.04-arm-small' || 'ubuntu-latest' }}
+    runs-on: ${{ matrix.arch == 'arm64' && 'pub-hk-ubuntu-24.04-arm-medium' || 'ubuntu-24.04' }}
```

Comment:
I've switched to medium since:
1. The cargo test invocation uses `--test-threads 16` which would benefit from more CPUs
2. It helps to offset the increased boot time of the custom runners (which don't benefit from the same large Slack pool as the standard runners)